components/prism-odin.js

```diff
@@ -0,0 +1,61 @@
+(function (Prism) {
+	var escapes = /\\(?:["'\\abefnrtv]|0[0-7]{2}|U[\dA-Fa-f]{6}|u[\dA-Fa-f]{4}|x[\dA-Fa-f]{2})/.source;
+
+	function withEscapes(pattern, flags) {
+		return RegExp(pattern.replace(/<escapes>/g, escapes), flags);
+	}
+
+	Prism.languages.odin = {
+		/**
+		 * The current implementation of multiline comments doesn't support nesting.
+		 */
+		'comment': [
+			{
+				pattern: /\/\*[\s\S]*?(?:\*\/|$)/,
```

Comment:
> No nested multiline comments :(

Turn the frown upside down :) Let's add support for 1 level of nested.

```suggestion
				pattern: /\/\*(?:[^/*]|\/(?!\*)|\*(?!\/)|\/\*[\s\S]*?(?:\*\/|$))*(?:\*\/|$)/,
```

Reply:
I should wrap my head around this pattern :)

Reply:
Whoops! Some checks were not successful. I should test this fix locally.

Reply:
It turns out that `[\s\S]*?` causes exponential backtracking. I’ve replaced it with `(?:\*(?!\/)|[^*])*`.

Reply:
Oh, you're right. My bad. Sorry.